app/predacons/src/predacons.py

```
@@ -425,18 +442,25 @@ def generate(*args, **kwargs):
         trust_remote_code = kwargs.get('trust_remote_code', False)
         apply_chat_template = kwargs.get('apply_chat_template',False)
-        temprature= kwargs.get('temprature',0.1)
+        temperature= kwargs.get('temperature',0.1)
+        stream = kwargs.get('stream',False)
         if apply_chat_template == True:
-            return Generate.generate_chat_output_from_model(model, tokenizer, sequence, max_length,temprature = temprature,trust_remote_code=trust_remote_code)
+            if stream:
+                return Generate.generate_chat_output_from_model_stream(model, tokenizer, sequence, max_length,temperature = temperature,trust_remote_code=trust_remote_code)
+            return Generate.generate_chat_output_from_model(model, tokenizer, sequence, max_length,temperature = temperature,trust_remote_code=trust_remote_code)
         try:
             if type(model) == torch._dynamo.eval_frame.OptimizedModule:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)